localization/ndt_scan_matcher/config/ndt_scan_matcher.param.yaml

```diff
@@ -41,10 +43,6 @@
     # Tolerance of distance difference between two initial poses used for linear interpolation. [m]
     initial_pose_distance_tolerance_m: 10.0
 
-    # neighborhood search method
-    # 0=KDTREE, 1=DIRECT26, 2=DIRECT7, 3=DIRECT1
-    neighborhood_search_method: 0
-
```

Comment:
@kminoda what reason did you delete other methods?

Reply:
This is because the proposed NDT method only supports Kd-tree and no longer supports DIRECT methods.

Reply:
I see. 
It might be better to describe it as a note in the PR description or README so that outsiders can understand it.

Reply:
I updated the description